psi4/src/read_options.cc

```diff
@@ -1400,6 +1400,10 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_int("DIIS_MAX_VECS", 10);
         /*- Do use DIIS extrapolation to accelerate convergence? -*/
         options.add_bool("DIIS", true);
+        /*- Do use a level shift? -*/
+        options.add_double("LEVEL_SHIFT", 0.0);
+        /*- DIIS error at which to stop applying the level shift -*/
+        options.add_double("LEVEL_SHIFT_CONVERGENCE", 1e-2);
```

Comment:
I think call it `LEVEL_SHIFT_CUTOFF` for consistency. https://psicode.org/psi4manual/master/manage_addon.html?highlight=threshold#how-to-name-keywords-in-psi4-src-read-options-cc

Reply:
Renamed.

Reply:
Tests pass again